CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Bug fixes
+* Modifying basis handling in `skbio.stats.composition.ilr_inv` prior to checking for orthogonality.  Now the basis is strictly assumed to be in the Aitchison simplex. ([#1393](https://github.com/biocore/scikit-bio/issues/1393))
```

Comment:
Won't have a chance to really review until the morning, but real quick comment:
This should be in the section just above (e.g. in `Backward-incompatible changes [experimental]`) as code that was already accounting for the inverse basis will need to be adjusted.
Thanks!